graylog2-web-interface/package.json

```diff
@@ -99,6 +101,6 @@
     "url-loader": "^0.5.6",
     "webpack": "^1.12.6",
     "webpack-dev-server": "^1.12.0",
-    "webpack-merge": "^0.7.x"
+    "webpack-merge": "^0.x.x"
```

Comment:
Due to its very limited usage and small api (we are consuming a single function) it should be okay.